hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/classic/PartitionSpecificCallable.java

```diff
@@ -30,8 +30,8 @@ public int getPartitionId() {
     public void run() {
         try {
             result = call();
-        } catch (Throwable e) {
-
+        } catch (Throwable t) {
+            throwable = t;
```

Comment:
I hope this change is okay. Just saw that `throwable is not set anywhere and it is private.